app/javascript/providers/StudySearchProvider.js

```diff
@@ -102,9 +102,10 @@ export function PropsStudySearchProvider(props) {
 
     const results = await fetchSearch('study', params)
 
+
     setSearchState({
       params,
-      isError: false,
+      isError: (results.ok || (results.ok == undefined))? false: !results.ok,
```

Comment:
At a glance I'd lean towards `!results.hasOwnProperty('ok')` over `results.ok == undefined`, but I can't actually think of a concrete case where we'd encounter this condition, and my naive check shows your updated code works as is.  So I consider the blocker resolved.

A minor non-blocking suggestion for conventional spacing around the ternary operators:
```suggestion
      isError: (results.ok || (results.ok == undefined)) ? false : !results.ok,
```